README.md

```diff
@@ -242,10 +436,9 @@ this repository.
 
 ## Docker
 
-The easiest way to setup this engine is within a Docker image. There is an
-included `Dockerfile` that can be used to build the Docker image. With the
-Docker daemon installed and running, navigate to the `REISE.jl` folder
-containing the `Dockerfile` and build the image:
+The easiest way to setup this engine is within a Docker image. Note, however, that the Docker image uses a [Gurobi Cloud License] and may not be able to handle larger simulations.
```

Comment:
I would be bit more specific in this sentence regarding machine requirements in order to run larger simulations.
@danielolsen do you have a good formulation for that?

Reply:
I agree. We probably want to say here that it requires significant resources to run a scenario in the U.S electric grid over a full year:
* x hours on y CPU
* z GB of RAM
* f GB of disk space to store the outputs of the simulation

Reply:
I don't have a good formula. RAM requirements will vary based on the interval length, total CPU-hours should be fairly similar, outputfile sizes can be pretty precise. I'm not running any USA scenarios right now so I don't have any `top` outputs to share.

I would think that RAM requirements are the most significant constraint, and that's more about interval length than the fact that it's a Docker image or that it uses a cloud license.

Reply:
Based on the Western scenarios that I have running now, and my recollection of the USA ones run recently, I would estimate that 1-2 GB of RAM are needed for each time interval in a USA run. E.g. a 24-hour interval simulation would require 24-48 GB; if only 16 GB RAM are available, I would recommend a time interval of 8 hours or less.

Reply:
This makes sense. I saw the memory constraints when attempting some larger simulations in the container, but if the container/cloud architecture isn't the issue, I removed any mention of how Docker/the cloud license may constrain the simulation size.

Instead, I added a `System Requirements` Section in the top `Dependencies` section with Daniel's guidance on how to estimate the amount of RAM needed for various simulations.